### ISSUE TITLE
yolo predict: InductorError in ultralytics/.venv/Lib/site-packages/torch/nn/modules/conv.py:_conv_forward

### ISSUE THREAD

**Issue by UltralyticsAssistant** [ultralytics/ultralytics#25119]
Automated fuzzing found a reproducible failure (confirmed 2/2 runs).

### Reproduce

```bash
yolo predict segment model=yolo26n-seg.pt imgsz=32 source=D:\a\ultralytics\ultralytics\ultralytics\assets\bus.jpg compile=max-autotune
```

### Details

- Outcome: `bug-candidate` · strategy: `invalid` · task: `segment` · seed: `29177681252`
- Run: https://github.com/ultralytics/ultralytics/actions/runs/29177681252

### Traceback (tail)

```
    return self.model(im, augment=augment, visualize=visualize, embed=embed, **kwargs)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\a\ultralytics\ultralytics\ultralytics\nn\tasks.py", line 154, in forward
    return self.predict(x, *args, **kwargs)
           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "D:\a\ultralytics\ultralytics\ultralytics\nn\tasks.py", line 171, in predict
    return self._predict_once(x, profile, visualize, embed)
           ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\a\ultralytics\ultralytics\ultralytics\nn\tasks.py", line 193, in _predict_once
    x = m(x)  # run
  File "D:\a\ultralytics\ultralytics\ultralytics\nn\modules\block.py", line 310, in forward
    y = list(self.cv1(x).chunk(2, 1))
             ~~~~~~~~^^^
  File "D:\a\ultralytics\ultralytics\.venv\Lib\site-packages\torch\nn\modules\module.py", line 1778, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "D:\a\ultralytics\ultralytics\.venv\Lib\site-packages\torch\nn\modules\module.py", line 1789, in _call_impl
    return forward_call(*args, **kwargs)
  File "D:\a\ultralytics\ultralytics\ultralytics\nn\modules\conv.py", line 89, in forward_fuse
    return self.act(self.conv(x))
                    ~~~~~~~~~^^^
  File "D:\a\ultralytics\ultralytics\.venv\Lib\site-packages\torch\nn\modules\conv.py", line 565, in forward
    return self._conv_forward(input, self.weight, self.bias)
           ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\a\ultralytics\ultralytics\.venv\Lib\site-packages\torch\nn\modules\conv.py", line 560, in _conv_forward
    return F.conv2d(
    


Set TORCHDYNAMO_VERBOSE=1 for the internal stack trace (please do this especially if you're reporting a bug to PyTorch). For even more developer context, set TORCH_LOGS="+dynamo"
```

### Environment

```
OS: Windows-2025Server-10.0.26100-SP0
Environment: Windows
Python: 3.13.14
Install: git
Path: D:\a\ultralytics\ultralytics\ultralytics
RAM: 15.99 GB
Disk: 6.3/150.0 GB
CPU: AMD EPYC 7763 64-Core Processor
CPU count: 4
GPU: None
GPU count: None
CUDA: None
Package Info: {'numpy': '✅ 2.2.6>=1.23.0', 'matplotlib': '✅ 3.11.0>=3.3.0', 'opencv-python': '✅ 5.0.0.93!=4.13.0.90,>=4.6.0', 'pillow': '✅ 12.3.0>=7.1.2', 'pyyaml': '✅ 6.0.3>=5.3.1', 'requests': '✅ 2.34.2>=2.23.0', 'torch': '✅ 2.13.0+cpu!=2.4.0,>=1.8.0; sys_platform == "win32"', 'torchvision': '✅ 0.28.0+cpu>=0.9.0', 'psutil': '✅ 7.2.2>=5.8.0', 'polars': '✅ 1.42.1>=0.20.0', 'nvidia-ml-py': '✅ 13.610.43>=12.0.0', 'ultralytics-thop': '✅ 2.0.20>=2.0.18'}
GitHub Info: {'RUNNER_OS': 'Windows', 'GITHUB_EVENT_NAME': 'schedule', 'GITHUB_WORKFLOW': 'Fuzz', 'GITHUB_ACTOR': 'glenn-jocher', 'GITHUB_REPOSITORY': 'ultralytics/ultralytics', 'GITHUB_REPOSITORY_OWNER': 'ultralytics'}
```

**Comment by UltralyticsAssistant** [comment]
👋 Hello @UltralyticsAssistant, thank you for your interest in Ultralytics 🚀! This is an automated response, and an Ultralytics engineer will also assist soon.

We recommend visiting the [Docs](https://docs.ultralytics.com/) for new users, where you can find many [Python](https://docs.ultralytics.com/usage/python) and [CLI](https://docs.ultralytics.com/usage/cli) usage examples and where many common questions may already be answered.

Thank you for providing a reproducible command, traceback, and environment details. If this is a 🐛 Bug Report, please also provide or confirm a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example), including the complete error output and any relevant `TORCHDYNAMO_VERBOSE=1` or `TORCH_LOGS="+dynamo"` output, to help us debug the issue.

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes) and [Tasks](https://docs.ultralytics.com/tasks) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by glenn-jocher** [comment]
Fixed by #25115, which moved the CPU compiler prerequisite check into `attempt_compile()` before PyTorch enters lazy Inductor compilation. This report came from fuzz run 29177681252 before that PR merged.
